client/components/WidgetComponent.js

```diff
@@ -3,7 +3,7 @@ import '../../style/widgets.scss'
 import { widgetsHost } from '../config'
 
 const WIDGET_URLS = {
-  timeline: `${widgetsHost}/timeline-app.html?useMockups`,
+  timeline: `${widgetsHost}/timeline-app.html?`,
   relations: `${widgetsHost}/relations-widget.html?useMockups`,
   map: `${widgetsHost}/map-app.html?`
```

Comment:
Shouldn't the URL be something like '/widgets/map-app.html'
I mean they are on the same server? And then it would also work locally.